packages/xo-server-backup-reports/src/index.js

```diff
@@ -188,15 +188,14 @@ class BackupReportsXoPlugin {
       formatDate,
     }
 
+    const backupReportTpl = log.data?.backupReportTpl ?? 'mjml'
```

Comment:
Declare it as a constant outside to avoid duplication.

Reply:
Do you mean declaring `const DEFAULT_TEMPLATE = 'mjml'` to avoid writing twice the same string, or passing `backupReportTpl` as a parameter to the `_metadataHandler` and `_vmHandler` functions to avoid writing it twice ?

Reply:
The first one